.pre-commit-config.yaml

```diff
@@ -17,21 +17,20 @@ repos:
         # Don't commit to main branch.
     -   id: no-commit-to-branch
 -   repo: https://github.com/psf/black
-    rev: '21.6b0'
+    rev: "21.6b0"
     hooks:
     -   id: black
         # Force black to run on whole repo, using settings from pyproject.toml
         pass_filenames: false
-        args: [--config=./pyproject.toml, .]
+        args: ["--config=./pyproject.toml", "--check", "docs", "lib", "noxfile.py", "setup.py"]
 -   repo: https://github.com/PyCQA/flake8
-    rev: '3.9.2'
+    rev: "3.9.2"
     hooks:
-        # Run flake8.
     -   id: flake8
-        args: [--config=./setup.cfg]
+        args: ["--config=./setup.cfg"]
```

Comment:
For consistency with `black` and `isort`: we should include a [`filename`](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-filename) key in `setup.cfg` to limit what is being tested, as we were doing previously using commands in `noxfile.py`.

Reply:
I'll check that, thanks

Reply:
same again, this is as frustrating as `black`, in that `filename` doesn't quite behave the same as `src_paths` from `isort`... so again we need to be explicit.